### ISSUE TITLE
Search should only return libraries results if there is an open doc

### ISSUE THREAD

**Issue by iwehrman** [adobe-photoshop/spaces-design#2211]
Trying to do anything with the results yields an unhandled exception.

**Comment by placegraphichere** [comment]
is this a technical limitation? There may be reasons to open library items with an empty doc state.

**Comment by chadrolfs** [comment]
If it's easy to do something useful with this, great, if not, disable it so it doesn't break.

**Comment by volfied** [comment]
When #2345 is merged, I will make changes to open graphics for edit, and filter out other assets from search.

**Comment by shaoshing** [comment]
In no-doc mode, search graphic asset and hit return will open the asset for edit. Please confirm @iwehrman

**Comment by iwehrman** [comment]
Wellll, it's almost fixed, except that we still return results in the search dialog for which we apparently do not support direct editing?
```
Executing action libraries.openGraphicForEdit after waiting 6ms; 1/0
[CC Lib] unsupported graphic type: image/png
```
How would it be hard to filter the results so that only the editable assets are listed?

**Comment by iwehrman** [comment]
The good news is that this is clearly no longer a high-priority issue, since there is no error and it works correctly much more often. Demoting to medium priority.

**Comment by iwehrman** [comment]
Fixed by #2427.
